PyDSS/dssInstance.py

```diff
@@ -394,16 +391,17 @@ def RunStep(self, step, updateObjects=None):
                 self._dssSolver.reSolve()
                 self._UpdatePlots()
                 if self._settings.exports.export_results:
-                    self.ResultContainer.UpdateResults()
+                    self.ResultContainer.UpdateResults(step)
```

Comment:
@nadiavp I think this change will work for you, in general. However, if you use the `get_dataframe` interface in pydss_results.py, you might see weird behavior because the pandas DataFrame will have indices with repeated values. I'm not sure how/whether they handle that.